cmd/microservice-sui-worker/README.md

```diff
@@ -0,0 +1,33 @@
+
+# microservice-sui-worker
+
+Receives decorated transfers, determines if they win, and pays them out if so!
```

Comment:
Would be worth talking about how this sends out transfer events as well. And why it does it (considering the classification circumstances.)